examples/shared_kv_storage/kv_storage/interface/shared_kv_interface.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from abc import ABC, abstractmethod
+
+class KvStorage(ABC):
+    """
+    KvStorage interface provides APIs to access key value store.
+    """
+    @abstractmethod
+    def open(self, lmdb_file, map_size="1 TB"):
+        """
+        Function to open the database file
+        Parameters:
+           - lmdb_file is the name and location of lmdb database file
+           - map_size is the maximum size of the database, it must be a multiple of the page size (4096)
+             and default to an insanely large max size (1 TB)
```

Comment:
Change "default" to "defaults"
Remove "insanely"

Reply:
Done.